bin/fin

```diff
@@ -1536,8 +1536,8 @@ show_help_list_commands_in_help ()
 	local executable="-executable"
 	is_mac && executable="-perm +100"
 
-	local addons_list=$(find "$addons_path" -type f ${executable} 2>/dev/null | sort -n 2>/dev/null)
-	if [[ "$addons_list" != "" ]]; then
+	local addons_list=$(find -L "$addons_path" -maxdepth 2 -type f ${executable} 2>/dev/null | sort -n 2>/dev/null)
+  if [[ "$addons_list" != "" ]]; then
```

Comment:
please revert breaking the indentation in this line. all indentation is currently tabs